iam.tf

```diff
@@ -56,7 +56,30 @@ resource "aws_iam_role_policy" "data_ingest_linux_parameter_store_iam_role" {
               "arn:aws:ssm:eu-west-2:*:parameter/mock_ftp_sftp_server_public_ip",
               "arn:aws:ssm:eu-west-2:*:parameter/mock_ftp_sftp_server_sftp_username"
             ]
-        }
+        },
+        {
+              "Effect": "Allow",
+              "Action": ["s3:ListBucket"],
+              "Resource": "${var.archive_bucket}"
+            },
+            {
+              "Effect": "Allow",
+              "Action": [
+                "s3:PutObject"
+              ],
+              "Resource": "${var.archive_bucket}/*"
+            },
+            {
+              "Effect": "Allow",
+              "Action": [
+                "kms:Encrypt",
+                "kms:Decrypt",
```

Comment:
if you can only put, do you need to be able to decrypt/reencrypt/describekey etc?

Reply:
co signed by David & AWS!